websites/R/Roblox Forum/presence.ts

```diff
@@ -0,0 +1,77 @@
+const presence = new Presence({
+		clientId: "993929577099448481",
+	}),
+	date = Date.now();
```

Comment:
```suggestion
	browsingTimestamp = Math.floor(Date.now() / 1000);
```

Reply:
dividing by 1000 is not necessary

Reply:
Not quite sure why the dividing by 1000 is necessary, but okay

Reply:
I'll look into how other presences are structured tomorrosw

Reply:
> Not quite sure why the dividing by 1000 is necessary, but okay

Like I said it is not. Discord accepts both timestamps in seconds and milliseconds

Reply:
right